docs/solidity-by-example.rst

```diff
@@ -377,29 +390,38 @@ high or low invalid bids.
         /// Reveal your blinded bids. You will get a refund for all
         /// correctly blinded invalid bids and for all bids except for
         /// the totally highest.
-        function reveal(uint[] _values, bool[] _fake,
-                        bytes32[] _secret)
+        function reveal(
+            uint[] _values,
+            bool[] _fake,
+            bytes32[] _secret
+        )
             onlyAfter(biddingEnd)
             onlyBefore(revealEnd)
         {
             uint length = bids[msg.sender].length;
-            if (_values.length != length || _fake.length != length ||
-                        _secret.length != length)
+            if (
+                _values.length != length
+                || _fake.length != length
```

Comment:
I would prefer the `||` to go to the previous line (similar to what would happen for a `,`), but I'm not sure if that is explicit in our coding style.